composer/2022_airflow_summit/data_analytics_process_test.py

```diff
@@ -125,7 +143,7 @@ def bq_dataset(test_bucket):
 
 # Retry if we see a flaky 409 "subnet not ready" exception
 @backoff.on_exception(backoff.expo, Aborted, max_tries=3)
-def test_process(test_bucket):
+def test_process(test_bucket, test_dataproc_batch):
```

Comment:
I am definitely rusty on my `pytest` but are you possibly creating two test buckets by referencing it in both `bq_dataset` and `test_process`?

Reply:
They are both defined as `scope="module"` so will only create the object once in this case.

Reply:
Great question! I am not because of the `scope=module` line - this tells `pytest` to only call that fixture once per module

Reply:
Yup! What Charlie said 😁